relays/ethereum/src/ethereum_client.rs

```diff
@@ -490,28 +581,31 @@ async fn submit_substrate_headers(
 	submitted_headers
 }
 
-/// Submit single Substrate header.
-async fn submit_substrate_header(
+/// Submit 4 Substrate headers in single PoA transaction.
+async fn submit_4_substrate_headers(
```

Comment:
Should this be changed to be something like `submit_batch_substrate_headers`, referring to `HEADERS_BATCH`?